subprojects/core/src/main/java/org/gradle/StartParameter.java

```diff
@@ -90,6 +81,7 @@
     private boolean continuous;
     private List<File> includedBuilds = new ArrayList<File>();
     private boolean buildScan;
+    private String bootstrapInitScriptUrl;
```

Comment:
I had originally tried to do this, but with a URL we don't have access to the dependency resolution stuff yet when the StartParameter is created. This object also seems to be more of a model without much of any logic in it and I didn't want to change that. We could change the list of files of init scripts to be a String or URL to we don't need any resolution logic but that's a change to the public (incubating) API. Thoughts?